src/newlines.cpp

```diff
@@ -782,6 +782,47 @@ void newline_del_between(chunk_t *start, chunk_t *end)
 } // newline_del_between
 
 
+void newlines_sparens()
+{
+   LOG_FUNC_ENTRY();
+
```

Comment:
Make a comment which link to meet the ROS2 style guide, as written at your #2360

Reply:
Done